osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -168,17 +200,22 @@ private double computeSpeedValue()
             // Scale the speed value with accuracy and OD.
             speedValue *= (0.95 + Math.Pow(Attributes.OverallDifficulty, 2) / 750) * Math.Pow(accuracy, (14.5 - Math.Max(Attributes.OverallDifficulty, 8)) / 2);
 
-            // Scale the speed value with # of 50s to punish doubletapping.
-            speedValue *= Math.Pow(0.98, countMeh < totalHits / 500.0 ? 0 : countMeh - totalHits / 500.0);
+            if (mods.Any(m => m is OsuModRelax))
+            {
+                // There isn't much we can do with scaling speed for Relax, so the minimum nerf is given.
+                speedValue *= 0.9;
+            }
+            else
+            {
+                // Scale the speed value with # of 50s to punish doubletapping.
+                speedValue *= Math.Pow(0.98, countMeh < totalHits / 500.0 ? 0 : countMeh - totalHits / 500.0);
+            }
 
             return speedValue;
         }
 
         private double computeAccuracyValue()
         {
-            if (mods.Any(h => h is OsuModRelax))
```

Comment:
Why? Accuracy pp works on the assumption that player tapped circles and with RX by it's logic it should always be 0

Reply:
You can still hit 100s and 50s in Relax without missing, and the old effective miss count including 100s/50s for Relax was removed, so accuracy PP is re-added and scaled as I believe that's a better idea. May need tweaking though.

Reply:
Again, accuracy pp is assuming that player is tapping the circles. Including relax here makes little sense from the concept of acc pp. I would rather prefer aim (or speed) accuracy penalties tweaked instead.

Reply:
They're already tweaked. I'll go ahead and remove accuracy PP and do more tweaks to the aim and speed formulas involving accuracy I guess.